src/haxelib/SemVer.hx

```diff
@@ -97,6 +97,10 @@ abstract SemVer(String) to String {
 	inline function get_valid()
 		return isValid(this);
 
+	@:op(a > b) static inline function greaterThan(a:SemVer, b:SemVer) {
```

Comment:
It feels a bit weird to only have `>` operator and not the other ones. For a single case, maybe it's simplier to just use `compare` method directly instead of `>`, even though it doesn't look great... I'm not sure though.